week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr[2]; // complete this statement
```

Comment:
You can't assume that the parameter `arr` will always have length 3. What would happen if there were fewer or more elements?